wiki/leave-insiders.md

```diff
@@ -0,0 +1,63 @@
+# Opting out of Windows Insider updates (Windows 11)
+
+If your machine is enrolled into the Windows Insider Program and you wish to stop receiving Insider updates, follow the steps below
+
+## Determine your current channel
+
+Open Settings, go to Windows Updates and Windows Insider Program
+
+![](img/leave-insiders/wu-page.png)
+
+There, look to the right side of "Choose your Insider settings" to determine your Insider channel, it can be Dev, Beta or Release Preview.
```

Comment:
```suggestion
Once there, look to the right side of *"Choose your Insider settings"* to see your Insider channel, It can be Dev, Beta or Release Preview.
```